code-of-conduct-ID.md

```diff
@@ -0,0 +1,40 @@
+## Pengantar
```

Comment:
hm, they're similar, but `Pendahuluan` sounds more correct for this, because the text in `## Introduction` section is more focus on the issue that discussed in this doc, while `Pengantar` not too focus usually.